examples/shortest_path_follower_example.py

```diff
@@ -62,7 +62,9 @@ def draw_top_down_map(info, heading, output_size):
 
```

Comment:
Are the changes to this file meant to stick or are they from testing?

Reply:
It was testing -> reverting.